### DISCUSSION TITLE
Native Aot for Avalonia.Xaml.Behaviors

### DISCUSSION THREAD

**Discussion by ChisterWu** [AvaloniaUI/Avalonia.Xaml.Behaviors#125] [Q&A]
I publish the app with Native aot, It will no work when  I use Avalonia.xaml.behaviors. I try to add  <Assembly Name="Avalonia.Xaml.Behaviors" Dynamic="Required All" /> in rd.xaml, but it  will failed to publish. How can i do if I wont to use  avalonia.xaml.behaviors

**Comment by wieslawsoltes** [comment]
Don't use Behaviors that rely on reflection.

**Comment by ChisterWu** [comment]
> Don't use Behaviors that rely on reflection.

Thank you
